frontend/web/index.html

```diff
@@ -34,7 +34,7 @@
 
   <script>
     // Actual URL is set at runtime with some nginx config level trickery.
-    window.testObserverAPIBaseURI = 'http://localhost:30000/';
+    window.testObserverAPIBaseURI = 'http://0.0.0.0:30000/';
```

Comment:
Why this change? [It seems like 0.0.0.0](https://superuser.com/questions/949428/whats-the-difference-between-127-0-0-1-and-0-0-0-0)) is `is commonly used as a non-routable meta-address used to designate an invalid, unknown or non applicable target (a no particular address placeholder).`, which makes sense to have as an "invalid until set" default, but for the the local dev stuff the default value is used (e.g. in firefox I see backend calls are made to `0.0.0.0:30000`, which admittingly I'm not sure why work).

Reply:
Ah, this is done in the Docker running backend end so that the service binds on all addresses, but is not actually needed in the frontend end, I'll revert this bit.